src/utils/ant.test.ts

```diff
@@ -6,65 +6,105 @@ import { sortANTRecords } from './ant.js';
 describe('sortANTRecordsByPriority', () => {
   it('should sort records by priority and then lexicographically', () => {
     const records = {
+      undername01: { priority: 1, transactionId: 'test', ttlSeconds: 1 }, // same priority, lexicographic sorting applied to the name
       undername1: { priority: 1, transactionId: 'test', ttlSeconds: 1 },
+      undername11: { priority: 1, transactionId: 'test', ttlSeconds: 1 }, // same priority, lexicographic sorting applied to the name
       undername2: { priority: 2, transactionId: 'test', ttlSeconds: 1 },
       undername3: { priority: 3, transactionId: 'test', ttlSeconds: 1 }, // colliding priorities default to lexicographic sorting
       undername4: { priority: 3, transactionId: 'test', ttlSeconds: 1 },
       undername5: { priority: 100, transactionId: 'test', ttlSeconds: 1 }, // priority does not represent the index or position of the record, just the order of resolution relative to other records
       noPriority: { transactionId: 'test', ttlSeconds: 1 },
+      noPriority01: { transactionId: 'test', ttlSeconds: 1 },
```

Comment:
Nit: Including a noPriority1 ensures that the numeric sorting properties are applied across both prioritized and non-prioritized entries.

Reply:
[e97e86f](https://github.com/ar-io/ar-io-sdk/pull/385/commits/e97e86ff2d9af06f481676d1410837681a79467e)